apps/easypid/app.config.js

```diff
@@ -1,180 +1,59 @@
-const { version } = require('./package.json')
-
-const APP_VARIANT = process.env.APP_VARIANT || 'production'
-
-const variants = {
-  development: {
-    bundle: '.dev',
-    name: ' (Dev)',
-    mediatorDid: 'did:web:mediator.dev.paradym.id',
-  },
-  preview: {
-    bundle: '.preview',
-    name: ' (Preview)',
-    mediatorDid: 'did:web:mediator.paradym.id',
-  },
-  production: {
-    bundle: '',
-    name: '',
-    mediatorDid: 'did:web:mediator.paradym.id',
-  },
+const { version: easypidVersion } = require('./package.json')
+const { version: paradymVersion } = require('../paradym/package.json')
```

Comment:
We'll get rid of the paradym app soon, so I wouldn't detect on it like this. Probably good to have a consistent version for both (and increasing one will increase the other, so we see it more as a codebase version). We can take the highest version of the two as a start now